### PR TITLE
File Added : Uploading asignment 2,3and4

### PR DESCRIPTION
**Pull request by arshad05786**

### CODE REVIEW

**Review by roshnidurugwar — APPROVED** [review]

**Review by Oshin-250197 — APPROVED** [review]

File: arshad_khan/Java_script/asignment2/findsecondlargestnumber.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                <button onclick="show()">Find Second Largest Number</button>
+                <h4>Find Second Largest Number</h4>
+                <p id="pid"></p>
+              </fieldset>
+            </div>
+          </div>
+        </div>
+      </center>
+    </body>
+</html>
```
Comment (Pawan1005) [inline arshad_khan/Java_script/asignment2/findsecondlargestnumber.html]: check indentation

File: arshad_khan/Java_script/asignment2/marksgrade.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            }
+        </style>
+    </head>
+    <body style="background-color:lightblue">
+        <center>
+              <div class="logincentrecard">
+                <div class="card">
+                  <div class="container">
+                    <h1>Find Result With Grade</h1>
+                    <fieldset style="width: 500px;height: 500px;background-color: rgb(182, 35, 157);>
```
Comment (Pawan1005) [inline arshad_khan/Java_script/asignment2/marksgrade.html]: check indentation

File: arshad_khan/Java_script/asignment2/marksgrade.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<input type="text" id="num5" placeholder="Enter Hindi Marks"><br><br>
+                      <button onclick="show()">Find Result With Grade</button>
+                      <h4>Find Result With Grade</h4>
+                      <p id="pid"></p>
+                  </div>
+                </div>
+            </div>
+        </center>
+    </body>
+</html>
```
Comment (Pawan1005) [inline arshad_khan/Java_script/asignment2/marksgrade.html:81]: check indentation

File: arshad_khan/Java_script/asignment3/fact.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+            <div class="logincentrecard">
+                <div class="card">
+                    <div class="container">
+                        <fieldset class="ft">
+                            <h1>FCTORIAL NUMBER</h1>
+                            <label for="Num1">Enter Number:</label>
+                            <input type="text" id="fact"
+                                placeholder="Enter Number">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br /><br />
+                            <button onclick="show()" style="background-color: red;color: white;">Find Factorial Number</button>
+                            <p id="pid"></p>
```
Comment (Pawan1005) [inline arshad_khan/Java_script/asignment3/fact.html:30]: check indentation

File: arshad_khan/Java_script/asignment3/leapyear.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            <div class="logincentrecard">
+                <div class="card">
+                    <div class="container">
+                        <fieldset class="ft">
+                            <h1>FIND LEAP YEAR</h1>
+                            <label for="Num1">Enter Year:</label>
+                            <input type="text" id="leap"
+                                placeholder="Enter year">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br /><br />
+                            <button onclick="show()" style="background-color: red;color: white;">Find Leap Year</button>
+                            <p id="pid"></p>
```
Comment (Pawan1005) [inline arshad_khan/Java_script/asignment3/leapyear.html:30]: check indentation

File: arshad_khan/Java_script/asignment3/pattern.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+                       <fieldset class="ft">
+                            <h1>PATTERN</h1>
+                            <label for="Num1">Enter Row:</label>
+                            <input type="text" id="row"
+                                placeholder="Enter Row">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br /><br />
+                                <label for="Num1">Enter Col:</label>
+                            <input type="text" id="col"
+                                placeholder="Enter col">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br /><br />
+                            <button onclick="show()" style="background-color: red;color: white;">Print Star</button>
+                            <p id="pid"></p>
```
Comment (Pawan1005) [inline arshad_khan/Java_script/asignment3/pattern.html]: check indentation

File: arshad_khan/Java_script/asignment4/automophic.html [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+        <center>
+            <div class="logincentrecard">
+                <div class="card">
+                    <div class="container">
+                            <h1>FIND AUTOMORPHIC NUMBER</h1>
+                            <label for="Num1">Enter Number:</label>
+                            <input type="text" id="num"
+                                placeholder="Enter Number">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br /><br />
+                            <button onclick="show()" style="background-color: red;color: white;">Find Automorphic</button>
+                            <p id="pid"></p>
```
Comment (Pawan1005) [inline arshad_khan/Java_script/asignment4/automophic.html]: check indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**